Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -50,7 +50,7 @@ protected function configureRoute(Route $route, \ReflectionClass $class, \Reflec
         // requirements (@Method)
         foreach ($this->reader->getMethodAnnotations($method) as $configuration) {
             if ($configuration instanceof Method) {
-                $route->setMethods(implode('|', $configuration->getMethods()));
+                $route->setMethods(explode('|', implode('|', $configuration->getMethods())));
```

Comment:
Normalize the array in https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/Configuration/Method.php#L46

?